code/datums/components/mood.dm

```diff
@@ -137,7 +137,7 @@
 				highest_absolute_mood = absmood
 
 	if(!conflicting_moodies.len) //no special icons- go to the normal icon states
-		if(sanity < 25)
+		if(sanity < 32)
```

Comment:
Why did you change this?

Reply:
Why is this a magic number in the first place?

Reply:
This is the threshold. I originally upped it to 30 because I also reduced the base modifier of ALL mood adjustments by 20%. In combination, that felt huge. 
However, the overall modifier is returned, so I felt I had more leeway

Reply:
Are you sure this is what you meant to do? Thats a < symbol not >

Reply:
Shouldn't this be `if(sanity < SANITY_INSANE)` ?

Reply:
Nice catch Qwerty. Changed that.